production/helm/templates/promtail/daemonset.yaml

```diff
@@ -48,8 +48,8 @@ spec:
               mountPath: /etc/promtail
             - name: varlog
               mountPath: /var/log
-            - name: varlibdockercontainers
-              mountPath: /var/lib/docker/containers
+            - name: dockercontainers
+              mountPath: {{ .Values.dataRoot }}/containers
```

Comment:
I don't see anything in the configmap, that load that path. I need to test it.

Reply:
Those log files are read by promtail indeed, those config just ensure promtail can reach those log files in container.